fix(sec): Upgrade jQuery to 3.5.1 + ACL form toggle resources

jdelpierre opened the pull request:
Upgraded jQuery library to version 3.5.1 to fix security.
Fixed a Javascript error on click events of checkboxes in ACL Resources forms (Hosts, HG, Services, SG).
It consists in showing/hiding the resource selection frames according to the status of the checkbox.

Refs: MON-6573

![image](https://user-images.githubusercontent.com/12846806/107710043-84aa2680-6cc6-11eb-9697-42cadb2c04fd.png)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [x] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

## Checklist

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

jdelpierre committed:
fix(sec): Upgrade jQuery to 3.5.1 + ACL form toggle resources

Upgraded jQuery library to version 3.5.1 to fix security.
Fixed a Javascript error on click events of checkboxes in ACL Resources forms (Hosts, HG, Services, SG).
It consists in showing/hiding the resource selection frames according to the status of the checkbox.

Refs: MON-6573

sc979 reviewed (commented):
Could you add the new lib ?

jdelpierre commented:
> Could you add the new lib ?

What do you mean? jQuery? 
I replaced the jquery.min.js with 3.5.1 and removed the non minified one because it is not used.

jdelpierre committed:
merge remote-tracking branch 'origin/master' into MON-6573-jquery-upgrade-fix-security

jdelpierre committed:
Merge remote-tracking branch 'origin/master' into MON-6573-jquery-upgrade-fix-security

jdelpierre committed:
Merge remote-tracking branch 'origin/master' into MON-6573-jquery-upgrade-fix-security

Jordan91 committed:
fix(UI): Add translation for acknowledge downtine popins, titles, notify option, date format and helper (#9503)

* MON-6235-Add-translation-for-Acknowledge-Downtine-popins-titles

* MON-6235-Add-translation-for-Acknowledge-Downtine-popins-titles-notify-helper

* MON-6235-Add-translation-for-Acknowledge-Downtine-popins-titles-notify-helper

* MON-6235-Add-translation-for-Acknowledge-Downtine-popins-titles-notify-helper

callapa committed:
fix(host template): Taking into account the request parameters for the list of host template (#9470)

Fixes: When calling the API to list host templates with a search parameter, an exception appears.

callapa committed:
enh(host group): take into account the ACL when displaying the list of host groups (#9488)

bdauria committed:
fix(Header/topCounter): Use new query parameter format for top counter resource-status links (#9490)

bdauria committed:
feat(Resource-status/Filter): store expand parameter in url and storage & fix multi-select criterias display issues (#9485)

kduret committed:
fix(test): fix acceptance test on lua broker output config

kduret committed:
Merge branch 'master' into MON-6573-jquery-upgrade-fix-security
